cmd/vmproxy/main.go

```diff
@@ -36,13 +33,13 @@ func main() {
 	var ln net.Listener
 
 	ln = must.Get(s.Listen("tcp", ":22"))
-	go sshServer(ln)
+	go vmproxy.SshServer(ln, vmName)
 
 	ln = must.Get(s.Listen("tcp", ":80"))
 	go promoteHTTPS(ln)
 
 	ln = must.Get(s.ListenTLS("tcp", ":443"))
-	vncProxy(s, ln)
+	vmproxy.VncProxy(s, ln, vncAddrPort)
```

Comment:
Capitalization of common initialisms
```suggestion
	vmproxy.VNCProxy(s, ln, vncAddrPort)
```